apps/openmw/mwrender/camera.cpp

```diff
@@ -98,6 +98,12 @@ namespace MWRender
         osg::Vec3d position = worldMat.getTrans();
         if (!isFirstPerson())
             position.z() += mHeight * mHeightScale;
+        else {
```

Comment:
It's OK to have the code formatted like this now, but when it's about to get merged it should be formatted according to [this](https://wiki.openmw.org/index.php?title=Code_Formatting_Conventions), i.e. opening/closing brackets at the same level, camelcase (use `positionBase` instead of `position_base` etc.).

Reply:
Ok